frontend/sozisel-app/src/components/utils/Table/Table.tsx

```diff
@@ -0,0 +1,167 @@
+import "./Table.scss";
```

Comment:
Did you copy that code from somewhere? Again some kind of abomination

Reply:
Yep -> https://material-ui.com/components/tables/#sorting-amp-selecting

Reply:
Ok, just add some comment that is copied from somewhere so nobody gets scared when they see this.

Reply:
I've already did that